app/src/main/java/com/github/se/icebreakrr/ui/sections/Notifications.kt

```diff
@@ -20,20 +32,43 @@ import com.github.se.icebreakrr.ui.navigation.NavigationActions
  */
 @SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
```

Comment:
I think that you should use the internal paddings of the Scaffold so that the screen adapts better to the screen size and the content doesn't overlap with the top and bottom bar.

Reply:
Yes indeed, you are rigth, but we could let that for the next sprint